src/easystar.js

```diff
@@ -119,6 +121,18 @@ EasyStar.js = function() {
     };
 
     /**
+    * Sets a directional condition on a tile
+    *
+    * @param {Number} x The x value of the point.
+    * @param {Number} y The y value of the point.
+    * @param {Array|String} list of all the allowed directions that can access
+    * the tile.
+    **/
+    this.setDirectionalCondition = function(x, y, allowedDirections) {
```

Comment:
We should create another API method for clearing directional conditions in the same manner as `stopAvoidingAdditionalPoint()`.